lib/srcmap.js

```diff
@@ -43,7 +43,7 @@ module.exports = {
     /**
       *  Takes a bytecode hexstring and returns a map indexed by offset
       *  that give the instruction number for that offset.
-      *
+      *ok
```

Comment:
?

Reply:
No doubt that was meant for some discord user -- I have too many Emacs windows.

Should be fixed in 6dc1164